python/packages/ai/teams/app.py

```diff
@@ -144,6 +150,56 @@ async def on_after_turn(context: TurnContext, state: TurnState):
         self._after_turn.append(func)
         return func
 
+    def error(self, func: Callable[[TurnContext, Exception], Awaitable[None]]):
+        """
+        Registers an error handler that will be called anytime
+        the app throws an Exception
+
+        ```python
+        # Use this method as a decorator
+        @app.error
+        async def on_error(context: TurnContext, err: Exception):
+            print(err.message)
+
+        # Pass a function to this method
+        app.error(on_error)
+        ```
+        """
+
+        self._error = func
+
+        if self._adapter:
+            self._adapter.on_turn_error = func
+
+        return func
+
+    async def process_activity(self, activity: Activity, auth_header: str) -> InvokeResponse:
```

Comment:
Seems this should be another feature that JS/C# should add. Can you help create new tasks for JS/C# SDK?